src/components/DesignTheme/View/Commons/PageHeader.jsx

```diff
@@ -41,20 +42,52 @@ const messages = defineMessages({
     id: 'minutes',
     defaultMessage: 'min',
   },
+  tipologia_persona: {
+    id: 'tipologia_persona',
+    defaultMessage: 'Tipologia persona',
+  },
+  ruolo: {
+    id: 'ruolo',
+    defaultMessage: 'Ruolo',
+  },
 });
 
 const PageHeader = props => {
   const intl = useIntl();
   return (
     <div className="row">
-      <div className="col-lg-8 px-lg-4 py-lg-2">
+      <div
+        className={`${
+          props.imageinheader ? 'col-lg-6' : 'col-lg-8'
+        } px-lg-4 py-lg-2`}
+      >
```

Comment:
Per fare questo solitamente usiamo la libreria [classnames](https://www.npmjs.com/package/classnames) in volto. Non che così non funzioni, ma uniformerei.
Esempio:

```javascript
import cx from classnames;

<div className={cx("px-lg-4 py-lg-2", {
  'col-lg-6': props.imageinheader,
  'col-lg-8': !props.imageinheader,
})}>
```